### ISSUE TITLE
Support for Windows users

### ISSUE THREAD

**Issue by GuillaumeBiessy** [doersino/nyum#10]
I really like the idea of hosting recipe on a website by using Markdown, keep up the good work !

I tried to follow the instructions and managed to found how to run bash on Windows through an Ubuntu distribution, however almost every line of code in build.sh throws an error, even the empty lines (because of the different line breaks I guess ?). I apologize if this sounds silly but I have very little experience on this topic.

Is this code intended to be used on a Windows machine ? Could you add instructions for Windows users ?

Thank you !

**Comment by doersino** [comment]
Hiya, thanks for checking this thing out! As mentioned in the readme, I haven't used Windows in a decade, so I don't have any experience with running bash scripts on Windows. (As a result, this code isn't really intended to be used on a Windows machine, but surely it's possible.)

It sounds like you've successfully set up WSL, so that seems like most of the work already done! Since you already suspect that line endings are to blame, it possible that you're running into [this issue](https://github.com/microsoft/WSL/issues/2318) where Git automatically converts Linux/macOS-style line breaks into Windows-style line breaks, which causes Bash to throw up? It sounds like you *might* need to run `git config --global core.autocrlf input`, then make a fresh clone of the repository and try running the script.

If that works for you, I'll link to this issue from the readme.

**Comment by GuillaumeBiessy** [comment]
Thank you for your quick reply.

I used the download ZIP approach rather than git clone and  RStudio as my editor.
The issue with line breaks was not initially there but occurred when I tried modifying and saving the file.

Thanks yo your answer, I was able to understand and fix the issue by going into the Rstudio Global Options => Code => Saving and setting Line ending conversion to LF.

I also got a second issue because I installed pandoc on Windows and not the virtual machine (I know I know but I'm not used to those things).
Now the bash build.sh command work for me and I was able to general the demo website, thank you !
